src/components/Faq.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import Expandable from './Expandable';
+import { faqQuestionsAnswers } from './Faq.content';
+import { FaqContainer } from './Faq.style';
+
+export default function Faq(): JSX.Element {
+	const [expanded, setExpanded] = useState<number | undefined>(undefined);
+
+	return (
+		<FaqContainer>
+			<h1>FAQ</h1>
```

Comment:
This should be FAQs (plural) as per the mock.

Reply:
[Updated](https://github.com/ardriveapp/price-calculator/commit/9268a6cb9e9bebfc75999c9e55fea8a36651668d)